test/test_gor4.py

```diff
@@ -0,0 +1,152 @@
+        result = gor4.predict(seq)
+
+        expected = 'CCCCCCCCCCHHHHHHHCCHHHHHHHHHHHCCCCEEEEECCEEEEEEEEC'
+        self.assertEqual(expected, result['predictions'])
+
+        # Below are the probabilities we get from running the command-line
+        # version of the GOR IV code (with the printout function adjusted
+        # to print 7 decimal places).
+        expected = (
+            (0.0000000, 0.0039858, 0.9960142),
+            (0.0000000, 0.0278830, 0.9721170),
+            (0.0000001, 0.0272379, 0.9727620),
+            (0.0000012, 0.1169491, 0.8830497),
+            (0.0001399, 0.0341633, 0.9656968),
+            (0.0017102, 0.0208701, 0.9774197),
+            (0.0105059, 0.0851670, 0.9043271),
+            (0.0555277, 0.0633068, 0.8811655),
+            (0.1095295, 0.0324757, 0.8579948),
+            (0.4153768, 0.0191681, 0.5654551),
+            (0.5536253, 0.1005364, 0.3458383),
+            (0.6340716, 0.1182708, 0.2476575),
+            (0.7778620, 0.0700642, 0.1520738),
+            (0.8236157, 0.0761598, 0.1002245),
+            (0.7757733, 0.0764664, 0.1477603),
+            (0.7078716, 0.0882713, 0.2038570),
+            (0.6208869, 0.0486473, 0.3304658),
+            (0.3671274, 0.0182623, 0.6146103),
+            (0.4309160, 0.0364063, 0.5326777),
+            (0.5071787, 0.1901496, 0.3026716),
+            (0.4161928, 0.4818794, 0.1019277),
+            (0.4236001, 0.5074177, 0.0689823),
+            (0.5267394, 0.4115629, 0.0616978),
+            (0.6260716, 0.2516718, 0.1222566),
+            (0.6345364, 0.2743220, 0.0911416),
+            (0.6292611, 0.2126158, 0.1581231),
+            (0.5834816, 0.0938588, 0.3226597),
+            (0.5517939, 0.0403667, 0.4078394),
+            (0.4721551, 0.0882974, 0.4395475),
+            (0.4599132, 0.0910693, 0.4490175),
+            (0.1938535, 0.0556542, 0.7504924),
+            (0.0518396, 0.0844042, 0.8637562),
+            (0.1642072, 0.0529393, 0.7828534),
+            (0.2086895, 0.3441093, 0.4472012),
+            (0.1385842, 0.6654806, 0.1959352),
+            (0.0953126, 0.6765780, 0.2281094),
+            (0.0322423, 0.9027448, 0.0650129),
+            (0.0787300, 0.6267382, 0.2945318),
+            (0.0786274, 0.5764974, 0.3448752),
+            (0.0711691, 0.1778753, 0.7509556),
+            (0.0765578, 0.1883003, 0.7351419),
+            (0.1004260, 0.7001448, 0.1994292),
+            (0.0736500, 0.8145193, 0.1118307),
+            (0.0321693, 0.8462602, 0.1215706),
+            (0.0968932, 0.6217796, 0.2813272),
+            (0.0629602, 0.7199594, 0.2170804),
+            (0.0186437, 0.5259689, 0.4553874),
+            (0.0009734, 0.5355605, 0.4634661),
+            (0.0000139, 0.8630118, 0.1369742),
+            (0.0000002, 0.9903608, 0.0096390))
+
+        for e, r in zip(expected, result['probabilities']):
+            self.assertAlmostEqual(e[0], r[0])
+            self.assertAlmostEqual(e[1], r[1])
+            self.assertAlmostEqual(e[2], r[2])
```

Comment:
Why can't you use ```assertEqual```, but make it so that the result also contains only seven decimal places? Otherwise, how should we know whether it's right?